src/main/java/com/cisco/trex/stateful/TRexAstfClient.java

```diff
@@ -350,7 +353,25 @@ private MetaData getAstfStatsMetaData() {
      */
     public LatencyStats getLatencyStats() {
         Map<String, Object> payload = this.createPayload();
-        return this.callMethod("get_latency_stats", payload, LatencyStats.class).get();
+        String json = this.callMethod("get_latency_stats", payload);
+        JsonElement response = new JsonParser().parse(json);
+        JsonElement latencyStatsData = response.getAsJsonArray().get(0).getAsJsonObject().get("result");
```

Comment:
confusing name with below java classes definition, can this be "latencyStatsJsonElement"?

Reply:
yes, I will rename this variable.